komodo/data/setup-py.sh

```diff
@@ -36,6 +36,10 @@ while test $# -gt 0; do
             shift
             export PYTHONPATH=$1
             ;;
+        --pip-build-opts)
+            shift
+            export PIP_BUILD_OPTS="--global-option=build_ext --global-option $1"
```

Comment:
Should be possible to define in the repository file.

Reply:
Yup, just had some problems with weird quotes appearing